### ISSUE TITLE
Installation Manager

### ISSUE THREAD

**Issue by pinpc** [platformio/platformio-core-installer#3026]
%23 Description of problem
  Leave a comment...

  BEFORE SUBMITTING, PLEASE SEARCH FOR DUPLICATES IN
  - https://github.com/platformio/platformio-vscode-ide/issues%3Fq=is%3Aissue 

  %23 Configuration

  VSCode: 1.80.0
  PIO IDE: v3.3.0
  System: Windows_NT, 10.0.19043, x64

  %23 Exception
  ```
  Error: Traceback (most recent call last):
  File "C:\Users\zhoup15\.platformio\.cache\tmp\get-platformio-1.2.0.py", line 71, in <module>
    main()
  File "C:\Users\zhoup15\.platformio\.cache\tmp\get-platformio-1.2.0.py", line 63, in main
    bootstrap()
  File "C:\Users\zhoup15\.platformio\.cache\tmp\get-platformio-1.2.0.py", line 49, in bootstrap
    pioinstaller.__main__.main()
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\pioinstaller\__main__.py", line 133, in main
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\core.py", line 1128, in __call__
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\core.py", line 1053, in main
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\core.py", line 1659, in invoke
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\core.py", line 1659, in invoke
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\core.py", line 1395, in invoke
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\core.py", line 754, in invoke
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\click\decorators.py", line 26, in new_func
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\pioinstaller\__main__.py", line 117, in core_check
  File "C:\Users\zhoup15\.platformio\.cache\tmp\.piocore-installer-uc4wdn6o\tmpgnz9_qdu\pioinstaller.zip\pioinstaller\core.py", line 169, in check
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\subprocess.py", line 488, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\subprocess.py", line 1207, in _execute_child
    startupinfo)
OSError: [WinError 1260] This program is blocked by group policy. For more information, contact your system administrator

	at c:\Users\zhoup15\.vscode\extensions\platformio.platformio-ide-3.3.0-win32-x64\node_modules\platformio-node-helpers\dist\index.js:1:5300
	at ChildProcess.i (c:\Users\zhoup15\.vscode\extensions\platformio.platformio-ide-3.3.0-win32-x64\node_modules\platformio-node-helpers\dist\index.js:1:4598)
	at ChildProcess.emit (node:events:513:28)
	at ChildProcess.emit (node:domain:489:12)
	at maybeClose (node:internal/child_process:1121:16)
	at ChildProcess._handle.onexit (node:internal/child_process:304:5)
  ```

**Comment by ivankravets** [comment]
Duplicate of https://github.com/platformio/platformio-core-installer/issues/3025
